packages/atomic/package.json

```diff
@@ -67,7 +65,7 @@
   "devDependencies": {
     "@axe-core/playwright": "4.9.1",
     "@babel/core": "7.24.5",
-    "@coveo/atomic": "2.65.5",
+    "@coveo/atomic": "file:.",
```

Comment:
Makes me wonder, why do we have a dev dependency on @coveo/atomic in the atomic package.json?

Reply:
Yes, we do for the tests/storybook